runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -239,4 +261,12 @@ default boolean swapDecant()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			position = 21,
+			keyName = "swapWildernessLever",
+			name = "Wilderness Lever",
+			description = "Swap wilderness lever left click to be Edgeville/Ardougne."
+	)
+	default boolean swapWildernessLever() { return true; }

```

Comment:
new lines

Reply:
Also this probably should not be true by default, not sure

Reply:
are you refering to the newlines on 271? If so, I formatted that default return to be like all the rest of them.

Reply:
on 371, { should go on separate line, same with }

Reply:
I mean 271 oops

Reply:
ok, do you want me to change all of the other lines like that in this file too while I'm in here? Because this one will be the odd one out if I make that change

Reply:
All other lines are fine, you probably just have the blocks collapsed in your IDE